### PR TITLE
Add more aathal comments

### PR DESCRIPTION
**Pull request by isuruamantha**

### CODE REVIEW

**Review by Arshardh — APPROVED** [review]

**Review by Arshardh — CHANGES_REQUESTED** [review]
format wrong in certain comments

File: comments.json [hunk 1]
```diff
@@ -80,119 +80,119 @@
     "comment": "shape"
```
Comment (Arshardh) [inline comments.json]: you have missed this

**Review by Arshardh — APPROVED** [review]

File: comments.json [hunk 2]
```diff
@@ -1,78 +1,198 @@
...
+    "awul_level": "shape",
+    "comment": "ඔරලෝසු නෑ..\nකියුටෙක්ස් නෑ..\nහෙයා ස්ටයිල් නෑ..\nමහලොකු මේකප් එකක් නෑ..\nදිලිසෙන චූටි බෑග් එකක් නෑ..\nඇස් කඩාගෙන යන පාට t සර්ට් එකක් නෑ..\nඒත් හැම කෙනෙක්ට ම වඩා ලස්සනයි.\nගොඩක් අය ආස\nඅන්න ඒ සරලකමට.\nබුකියේ දුටු තවත් ලස්සන චායාරූපයක් නලුවන්ට වැටෙන Like මේ වගේ අහිංසක මල් කැකුළු වලට වැටෙන්නෙ නෑ"
+  },
+  {
+    "awul_level": "poddak",
+    "comment": "කලක් තිස්සේ ලමා අපචාර පිළිබදව සැකපිට අත්තඩංගු⁣වේ පසුවූ \nමෙවැනි පුද්දලයන් නීති⁣ය රැහැනට අසුකොට නිසි දඩවම් ලබා දියයුතු කාලය එළඹ ඇත.. එවැනි තත්වටන් යටතේ මෙවැනි වූ පාපතරයන් බුකිය තුළ තම ඡායාරෑප දමා \nබුකියේ අසරණ ළමුන් අපහරණය කිරීමට විරුද්ධ වීම අප කාගෙත් යුතුකමක් මෙන්ම \nවගකීමක් ලෙසද අකමැත්තෙන් වුවද සැලකළ යුතුය.."
+  },
+  {
+    "awul_level": "shape",
+    "comment": "දැක්ක ගමන් හිතුවෙ කැළණි හිඟන්නෙක් කියල නේද. නෑ මේ ඉන්නෙ අර ඉතාලියේ ඉඳන් ඇවිල්ල නිරෝදායනය නොවී රට ඇතුලෙ හැංගිලා ඉන්න රට විරු😡 සහෝදරියක්. එයාපෝට් එකේදි කට තිබ්බට දැන්නම් මෙයා කාටවත් මූන දෙන්න බැරුව කැලේක හැංගිලාලු ඉන්නෙ. මෙයා ඉන්න තැනක් හරියටම දන්නව නම් ලඟම පොලීසියට කියල රටවෙනුවෙන් අපේ යුතුකම අපි ඉටු කරමු.",
```
Comment (amjadhifthikar) [inline comments.json]: "," at the end have to be removed.

Comment (amjadhifthikar) [reply comments.json]: Fix other places as well

**Review by amjadhifthikar — APPROVED** [review]

**Pull request merged**